pool/_RPC_api.gno

```diff
@@ -21,7 +21,7 @@ type RpcPool struct {
 	BalancesToken1 string `json:"balanceToken1"`
 
 	// fee is the fee tier of the pool
-	Fee uint16 `json:"fee"`
+	Fee uint32 `json:"fee"`
```

Comment:
What is the reason behind using uint32 for fees?

Reply:
Position contract uses uint32 as pool's fee data type.
However, it can be still `uint16` I think

Reply:
keeping `uint32` since most of other realm(position, staker) uses uint32